templates/zerver/app/index.html

```diff
@@ -248,7 +248,7 @@
                         <div id="loading_older_messages_indicator"></div>
                         <div id="page_loading_indicator"></div>
                         <div id="message_feed_errors_container"></div>
-                        <div class="message_table focused_table" id="zhome" role="list" aria-live="polite" aria-label="{{ _('Messages') }}">
+                        <div class="message_table focused-list" id="zhome" role="list" aria-live="polite" aria-label="{{ _('Messages') }}">
```

Comment:
`focused-message-list` might be worth it for specificity?

Reply:
Absolutely. Pushed fresh work with that change.